packages/design-system/src/components/link.tsx

```diff
@@ -0,0 +1,65 @@
+import { styled, theme } from "../stitches.config";
+import { textVariants } from "./text";
+
+export const Link = styled("a", {
+  cursor: "pointer",
+  variants: {
+    variant: {
+      inherit: {
+        fontFamily: "inherit",
+        fontWeight: "inherit",
+        fontSize: "inherit",
+        lineHeight: "inherit",
+        letterSpacing: "inherit",
+        textTransform: "inherit",
+        textIndent: "inherit",
+        textDecoration: "underline",
+      },
+      regular: textVariants.regularLink,
+      label: textVariants.labelLink,
+      mono: textVariants.monoLink,
+      monoBold: textVariants.monoBoldLink,
+    },
+    color: {
+      main: {
+        color: theme.colors.foregroundMain,
+        "&:hover, &:visited": { color: theme.colors.foregroundMain },
+      },
+      contrast: {
+        color: theme.colors.foregroundContrastMain,
+        "&:hover, &:visited": { color: theme.colors.foregroundContrastMain },
+      },
+      subtle: {
+        color: theme.colors.foregroundTextSubtle,
+        "&:hover, &:visited": { color: theme.colors.foregroundTextSubtle },
+      },
+      moreSubtle: {
+        color: theme.colors.foregroundTextMoreSubtle,
+        "&:hover, &:visited": { color: theme.colors.foregroundTextMoreSubtle },
+      },
+      inherit: {
+        color: "inherit",
+        "&:hover, &:visited": { color: "inherit" },
+      },
+    },
+    underline: {
+      none: {
+        textDecoration: "none",
+        "&:hover": { textDecoration: "none" },
+      },
+      hover: {
+        textDecoration: "none",
+        "&:hover": { textDecoration: "underline" },
+      },
+      always: {
+        textDecoration: "underline",
+        "&:hover": { textDecoration: "underline" },
+      },
+    },
```

Comment:
This better be decided on project level

Reply:
I checked all our usages, this works, also I stole this api from material ui https://mui.com/material-ui/react-link/#underline

I think we should be fine, this gives us more of a standard api for links

Reply:
the 3 underline behavior is all we ever gonna need